src/errors/runtime/message.js

```diff
@@ -18,29 +18,57 @@ export default {
     clientFunctionCantResolveTestRun:                               "{#instantiationCallsiteName} cannot implicitly resolve the test run in context of which it should be executed. If you need to call {#instantiationCallsiteName} from the Node.js API callback, pass the test controller manually via {#instantiationCallsiteName}'s `.with({ boundTestRun: t })` method first. Note that you cannot execute {#instantiationCallsiteName} outside the test code.",
     regeneratorInClientFunctionCode:                                `{#instantiationCallsiteName} code, arguments or dependencies cannot contain generators or "async/await" syntax (use Promises instead).`,
     invalidClientFunctionTestRunBinding:                            'The "boundTestRun" option value is expected to be a test controller.',
     invalidValueType:                                               '{smthg} is expected to be a {type}, but it was {actual}.',
     unsupportedUrlProtocol:                                         'The specified "{url}" test page URL uses an unsupported {protocol}:// protocol. Only relative URLs or absolute URLs with http://, https:// and file:// protocols are supported.',
-    unableToOpenBrowser:                                            'Was unable to open the browser "{alias}" due to error.\n\n{errMessage}',
     testControllerProxyCantResolveTestRun:                          `Cannot implicitly resolve the test run in the context of which the test controller action should be executed. Use test function's 't' argument instead.`,
-    requestHookConfigureAPIError:                                   'There was an error while configuring the request hook:\n\n{requestHookName}: {errMsg}',
     timeLimitedPromiseTimeoutExpired:                               'Timeout expired for a time limited promise',
-    forbiddenCharatersInScreenshotPath:                             'There are forbidden characters in the "{screenshotPath}" {screenshotPathType}:\n {forbiddenCharsDescription}',
     cantUseScreenshotPathPatternWithoutBaseScreenshotPathSpecified: 'Cannot use the screenshot path pattern without a base screenshot path specified',
+    cantSetVideoOptionsWithoutBaseVideoPathSpecified:               'Unable to set video or encoding options when video recording is disabled. Specify the base path where video files are stored to enable recording.',
     multipleAPIMethodCallForbidden:                                 'You cannot call the "{methodName}" method more than once. Pass an array of parameters to this method instead.',
-    invalidReporterOutput:                                          "Specify a file name or a writable stream as the reporter's output target."
+    invalidReporterOutput:                                          "Specify a file name or a writable stream as the reporter's output target.",
+
+    cantReadSSLCertFile: 'Unable to read the "{path}" file, specified by the "{option}" ssl option. Error details:\n' +
```

Comment:
somewhere we have `cant` and somewhere `cannot`, let's choose one form

Reply:
let it be `cannot`